bundle/manifests/aws-nlb-helper-operator-controller-manager-metrics-service__service.yaml

```diff
@@ -0,0 +1,16 @@
+kind: Service
+metadata:
+  creationTimestamp: null
+  labels:
+    control-plane: controller-manager
+  name: aws-nlb-helper-operator-controller-manager-metrics-service
```

Comment:
Why there are 2 services on bundle directory managing the same metrics service?:
- bundle/manifests/aws-nlb-helper-operator-controller-manager-metrics-service__service.yaml
- bundle/manifests/aws-nlb-helper-operator-controller-manager-metrics-monitor_monitoring.coreos.com_v1_servicemonitor.yaml


Maybe..
- after doing so many changes, you should first remove the bundle dir, to see if current source creates a single Service (bundle never remove not used old trash)?
- It is a config error?